corehq/apps/export/views.py

```diff
@@ -1406,16 +1379,12 @@ def fmt_export_data(self, export):
             'emailedExport': emailed_export,
             'editUrl': reverse(EditNewCustomFormExportView.urlname,
                                args=(self.domain, export.get_id)),
-            'downloadUrl': self._get_download_url(export.get_id, is_legacy),
+            'downloadUrl': self._get_download_url(export.get_id),
```

Comment:
Stylistic comment: I'd get rid of `_get_download_url` altogether and just inline the `reverse` call.